README.md

```diff
@@ -0,0 +1,58 @@
+# Awesome books: plain JavaScript with classes
+
+- In this project, we build a basic website that allows users to add/remove books from a list. 
+- We achieve that by using JavaScript classes and arrays. 
+- We also dynamically modify the DOM and added basic events.
+
+
+
+## Built With
+- Major languages (HTML, CSS , JavaScript)
+- Frameworks (Bootstrap)
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+**This is the Awesome books: plain JavaScript with classes**
+
+
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
- [ ] Please remove the repetition of sentences that are shown here. As best practice, we are expected not to repeat the same thing more often because it brings in confusion and complexity 😎